_includes/scenarios/integrate/requests-over-templates.md

```diff
@@ -139,7 +163,7 @@ If you synchronize the Templator Connector (`POST /api/v2/Account/Sync`) you wil
       "request": {
         "createdBy": "id1...",
         "createdByDevice": "DVC...",
-        "createdAt": "2022-11-04T13:31:01.360Z",
+        "createdAt": "<time-of-creation>",
```

Comment:
We should omit the hyphens here, as we do not use them for other placeholders or example values either.